src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,221 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushSelected")]
+						public static ResourceValue<Brush> Selected => new("PipsPagerSelectionIndicatorBorderBrushSelected", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "PipsPagerSelectionIndicatorBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("PipsPagerSelectionIndicatorBorderBrushDisabled", true);
+					}
+
+					public static class VisualStateEllipse
+					{
+						public static class Fill
+						{
+							[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerVisualStateEllipseFill")]
+							public static ResourceValue<Brush> Default => new("MaterialPipsPagerVisualStateEllipseFill", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerVisualStateEllipseFillPointerOver")]
+							public static ResourceValue<Brush> PointerOver => new("MaterialPipsPagerVisualStateEllipseFillPointerOver", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerVisualStateEllipseFillPressed")]
+							public static ResourceValue<Brush> Pressed => new("MaterialPipsPagerVisualStateEllipseFillPressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerVisualStateEllipseFillFocused")]
+							public static ResourceValue<Brush> Focused => new("MaterialPipsPagerVisualStateEllipseFillFocused", true);
+						}
+					}
+
+					[ResourceKeyDefinition(typeof(double), "PipsPagerHorizontalOrientationButtonWidth")]
+					public static ResourceValue<double> HorizontalOrientationWidth => new("PipsPagerHorizontalOrientationButtonWidth", true);
+
+					[ResourceKeyDefinition(typeof(double), "PipsPagerVerticalOrientationButtonWidth")]
+					public static ResourceValue<double> VerticalOrientationWidth => new("PipsPagerVerticalOrientationButtonWidth", true);
+
+					[ResourceKeyDefinition(typeof(double), "PipsPagerHorizontalOrientationButtonHeight")]
+					public static ResourceValue<double> HorizontalOrientationHeight => new("PipsPagerHorizontalOrientationButtonHeight", true);
+
+					[ResourceKeyDefinition(typeof(double), "PipsPagerVerticalOrientationButtonHeight")]
+					public static ResourceValue<double> VerticalOrientationHeight => new("PipsPagerVerticalOrientationButtonHeight", true);
+
+					[ResourceKeyDefinition(typeof(Thickness), "PipsPagerButtonBorderThickness")]
+					public static ResourceValue<Thickness> BorderThickness => new("PipsPagerButtonBorderThickness", true);
+				}
+
+				public static class MainEllipse
+				{
+					public static class Size
+					{
+						[ResourceKeyDefinition(typeof(double), "PipsPagerNormalEllipseSize")]
+						public static ResourceValue<double> Default => new("PipsPagerNormalEllipseSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "PipsPagerSelectedEllipseSize")]
+						public static ResourceValue<double> Selected => new("PipsPagerSelectedEllipseSize", true);
+					}
+				}
```

Comment:
Let's just do this as `PipsPager.Resources.NormalEllipseSize` and `PipsPager.Resources.SelectedEllipseSize`